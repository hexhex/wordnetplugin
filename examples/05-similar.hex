%% this returns all words, similar to a word
%% POS: adjectives

%% the search number for similar is 5

word("bright").
similar(W,SW,S,SS) :- &wordnet[5,W](_,SW,S,SS), word(W).

similar("bright",1,"silvery",18), 
similar("bright",1,"iridescent",12), 
similar("bright",1,"shining",11), 
similar("bright",1,"shiny",11), 
similar("bright",1,"brilliant",8), 
similar("bright",3,"intelligent",1), 
similar("bright",6,"glorious",1)?
