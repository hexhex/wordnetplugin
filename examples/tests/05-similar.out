{similar("bright",1,"twinkling(a)",19), similar("bright",1,"silvery",18), similar("bright",1,"silvern",18), similar("bright",1,"silver",18), similar("bright",1,"shimmery",17), similar("bright",1,"self-luminous",16), similar("bright",1,"slick",15), similar("bright",1,"silklike",15), similar("bright",1,"silky",15), similar("bright",1,"silken",15), similar("bright",1,"sleek",15), similar("bright",1,"satiny",15), similar("bright",1,"noctilucent",14), similar("bright",1,"lurid",13), similar("bright",1,"pearlescent",12), similar("bright",1,"opaline",12), similar("bright",1,"opalescent",12), similar("bright",1,"nacreous",12), similar("bright",1,"iridescent",12), similar("bright",1,"shining",11), similar("bright",1,"shiny",11), similar("bright",1,"sheeny",11), similar("bright",1,"lustrous",11), similar("bright",1,"glossy",11), similar("bright",1,"glistening",11), similar("bright",1,"glimmery",10), similar("bright",1,"ardent",9), similar("bright",1,"brilliant",8), similar("bright",1,"bright_as_a_new_penny(p)",7), similar("bright",1,"glary",6), similar("bright",1,"glaring",6), similar("bright",1,"fulgent",6), similar("bright",1,"dazzling",6), similar("bright",1,"blinding",6), similar("bright",1,"blazing",6), similar("bright",1,"refulgent",5), similar("bright",1,"radiant",5), similar("bright",1,"effulgent",5), similar("bright",1,"beamy",5), similar("bright",1,"beaming",5), similar("bright",1,"buttonlike",4), similar("bright",1,"buttony",4), similar("bright",1,"beadlike",4), similar("bright",1,"beady",4), similar("bright",1,"sparkly",3), similar("bright",1,"scintillating",3), similar("bright",1,"scintillant",3), similar("bright",1,"glittery",3), similar("bright",1,"glittering",3), similar("bright",1,"glistering",3), similar("bright",1,"glinting",3), similar("bright",1,"fulgid",3), similar("bright",1,"coruscant",3), similar("bright",1,"aglitter(p)",3), similar("bright",1,"luminous",2), similar("bright",1,"lucent",2), similar("bright",1,"lambent",2), similar("bright",1,"aglow(p)",2), similar("bright",1,"nitid",1), similar("bright",1,"gleaming",1), similar("bright",1,"agleam",1), similar("bright",2,"colourful",1), similar("bright",2,"colorful",1), similar("bright",3,"intelligent",1), similar("bright",4,"light",1), similar("bright",5,"polished",1), similar("bright",6,"glorious",1), similar("bright",8,"reverberant",1), similar("bright",9,"happy",1), similar("bright",10,"auspicious",1), word("bright")}
